ddtrace/internal/telemetry/writer.py

```diff
@@ -502,6 +500,51 @@ def add_log(self, level, message, stack_trace="", tags=None):
             # Logs are hashed using the message, level, tags, and stack_trace. This should prevent duplicatation.
             self._logs.add(data)
 
+    def add_integration_error_log(self, msg, exc, warning=False):
```

Comment:
nit: can we define two methods: `add_error_log`  and `add_warning_log`?

Reply:
Just to be sure, the telemetry log needs to have an `error` level in order to appear in error tracking. So the only difference is about the log call. Do we agree that the definition of two methods would only serve the log.debug/log.warning or did you want to do that for another reason ?

Reply:
Resolved as warning logs will be removed from contrib/ (as we cannot prevent them to be seen by a customer by default without adding configuration).